test/e2e/framework/util.go

```diff
@@ -1198,10 +1200,10 @@ func hasRemainingContent(c clientset.Interface, clientPool dynamic.ClientPool, n
 		obj, err := dynamicClient.Resource(&apiResource, namespace).List(metav1.ListOptions{})
 		if err != nil {
 			// not all resources support list, so we ignore those
-			if apierrs.IsMethodNotSupported(err) || apierrs.IsNotFound(err) || apierrs.IsForbidden(err) {
-				continue
+			if !apierrs.IsMethodNotSupported(err) && !apierrs.IsNotFound(err) && !apierrs.IsForbidden(err) {
+				Logf("namespace: %s, resource: %v, unable to list resources, error: %v", namespace, apiResource.Name, err)
```

Comment:
Since you inserted `WaitForServerPreferredNamespacedResources` earlier, do you still need these changes?

Reply:
Metrics server may still be down after that point?
I'm fine with keeping the error, and fix it if there is flake on this line.

Reply:
Done